benchmark.js

```diff
@@ -47,4 +47,16 @@ suite('chalk', () => {
 	bench('cached: 1 style nested non-intersecting', () => {
 		chalkBgRed(blueStyledString);
 	});
+
+	set('iterations', 10000);
```

Comment:
Why manually set iterations here? This can skew results if too low.

Reply:
Parsing strings for the special `{bold ...}`-syntax is a lot slower.
Benchmark 'cached: 1 style' is almost 200 times faster than 'cached: 1 style template literal'. So I reduced the number of iterations accordingly.

Better than specifying the number of iterations would be to set a minimal runtime for each benchmark. So instead of
https://github.com/chalk/chalk/blob/26c73d2b80f06f5673a7bb44433f69316f229629/benchmark.js#L6
something like
```
set('mintime', 500); 
```

But this feels out of the scope of this PR.

Reply:
> Better than specifying the number of iterations would be to set a minimal runtime for each benchmark.

I agree. Mind opening a PR? :)